issuer.go

```diff
@@ -0,0 +1,111 @@
+package issuer
+
+import (
+	"bytes"
+	"compress/flate"
+	"crypto/ecdsa"
+	"encoding/json"
+	"github.com/skip2/go-qrcode"
+	"gopkg.in/square/go-jose.v2"
+	"strconv"
+	"time"
+)
+
+const (
+	MAX_SINGLE_JWS_SIZE = 1195
+	MAX_CHUNK_SIZE      = 1191
+)
+
+type SmartHealthCard struct {
+	IssuerUrl            string                 `json:"iss"`
+	IssuanceDate         int                    `json:"nbf"`
+	VerifiableCredential map[string]interface{} `json:"vc"`
+}
+
+type IssueCardInput struct {
+	IssuerUrl            string
+	PrivateKey           *ecdsa.PrivateKey
+	KeyId                string
+	VerifiableCredential map[string]interface{}
+}
+
+func IssueCard(input IssueCardInput) (string, error) {
+	card := SmartHealthCard{
+		IssuerUrl:            input.IssuerUrl,
+		IssuanceDate:         time.Now().Hour(),
+		VerifiableCredential: input.VerifiableCredential,
+	}
+
+	jws, err := card.Sign(input.PrivateKey, input.KeyId)
+	if err != nil {
+		return "", err
```

Comment:
Can we use `fmt.Errorf` to add some context to errors? I think it might be confusing for users to only see the original error (e.g., from `NewSigner`).